From andrea.gelmini@gelma.net  Fri Feb 26 16:42:43 2010
From: Andrea Gelmini <andrea.gelmini@gelma.net>
Date: Fri, 26 Feb 2010 17:36:58 +0100
Subject: Staging: arlan: arlan.h: more Checkpatch cleanup
To: gregkh@suse.de
Cc: devel@driverdev.osuosl.org, Andrea Gelmini <andrea.gelmini@gelma.net>, bzolnier@gmail.com
Message-ID: <1267202220-15640-3-git-send-email-andrea.gelmini@gelma.net>


ERROR: do not use C99 // comments
+      volatile	u_char radioModule;// shows EEPROM, can be overridden at 0x111

ERROR: do not use C99 // comments
+      volatile	u_char defaultChannelSet; // shows EEProm, can be overriiden at 0x10A

Signed-off-by: Andrea Gelmini <andrea.gelmini@gelma.net
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/arlan/arlan.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/arlan/arlan.h
+++ b/drivers/staging/arlan/arlan.h
@@ -90,8 +90,8 @@ struct arlan_shmem {
       volatile	u_char hardwareType;
       volatile	u_char majorHardwareVersion;
       volatile	u_char minorHardwareVersion;
-      volatile	u_char radioModule;// shows EEPROM, can be overridden at 0x111
-      volatile	u_char defaultChannelSet; // shows EEProm, can be overriiden at 0x10A
+      volatile	u_char radioModule;/* shows EEPROM, can be overridden at 0x111 */
+      volatile	u_char defaultChannelSet; /* shows EEProm, can be overriiden at 0x10A */
       volatile	u_char _2[47];
 
       /* Control/Status Block - 0x0080 */
